types/react/index.d.ts

```diff
@@ -2321,7 +2322,7 @@ declare namespace React {
         filterUnits?: number | string;
         floodColor?: number | string;
         floodOpacity?: number | string;
-        focusable?: number | string;
+        focusable?: Booleanish;
```

Comment:
https://www.w3.org/TR/SVGTiny12/interact.html#focusable-attr

```suggestion
        focusable?: Booleanish | "auto";
```

Reply:
I like it, but I don't have permission to update your fork.  Please go ahead and add it to the PR.